_sass/pages/_not_found.scss

```diff
@@ -14,41 +11,43 @@ body.not-found {
   }
 
   .site-wrapper {
-    display: table;
-    height: 100%;
-    min-height: 100%;
-    width: 100%;
-  }
-
-  .site-wrapper-inner {
-    display: table-cell;
-    vertical-align: top;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;;
   }
 
   .cover-container {
-    margin-left: auto;
-    margin-right: auto;
+    margin: auto;
+    max-width: 600px;
```

Comment:
Is this width within a media query? Seems like it would be too wide for mobile.

Reply:
this is not within a media query. This was called out in the design specs.